engine-src/Game/LambdaHack/Common/ReqFailure.hs

```diff
@@ -147,9 +151,11 @@ showReqFailure reqFailure = case reqFailure of
   AlterBlockActor -> "blocked by an actor"
   AlterBlockItem -> "jammed by an item"
   AlterNothing -> "wasting time on altering nothing"
-  CloseDistant -> "trying to close a dinstant object"
+  CloseDistant -> "trying to close a distant position"
```

Comment:
I said "position" myself, but now I think it's wrong. I'd put "terrain" or "at distant position".